core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -215,6 +215,10 @@ final case class EitherT[F[_], A, B](run: F[A \/ B]) {
 }
 
 object EitherT extends EitherTInstances {
+  def F[F[_], A, B](a: F[A \/ B]): EitherT[F, A, B] = apply(a)
+  def A[F[_]: Applicative, A, B](a: A): EitherT[F, A, B] = apply(Applicative[F].point(-\/(a)))
```

Comment:
`pureLeft`?

Reply:
Or just `left` and rename the current `left` to `leftT`, which would make it consistent with `eitherT`. (And address 7.3.x).

Reply:
cats just did this dance, I'm not so sure we want to go changing the meaning of existing methods... it's a massive pain on existing codebases with almost zero payback.

Reply:
I find methods called `A`, `B`, `F` pretty terrible. Especially if the names of the type parameters are arbitrary and usually don't match the names on the use site.

Reply:
I'll change to `pure{Left, Right}`, that sounds good. I'm not wedded to the names. But I'm not keen on changing existing methods. Any other suggestions for `fromDisjunction` ? It's such a long easily mis-spelt name.

Reply:
`either`?